.buildkite/scripts/lib/BUILD

```diff
@@ -1,5 +1,5 @@
 shell_library()
 
 shunit2_tests(
-    name="tests"
+    name="tests",
 )
```

Comment:
okay, yes, this is not a required change, but i'd prefer to not run CI again if i don't need to lol